ISSUE-67, update to tournament-v5

happsie opened the pull request:
Resolves #67

KnutZuidema commented on riot/lol/constants.go:
```diff
@@ -38,14 +38,14 @@ const (
+	endpointTournamentBase                     = endpointBase + "/tournament/v5"
 	endpointCreateTournamentCodes              = endpointTournamentBase + "/codes?count=%d&tournamentId=%d"
-	endpointGetLobbyEvents                     = endpointTournamentBase + "/lobby-events/by-code/%s"
+	endpointGetLobbyEvents                     = endpointTournamentBase + "/lobby-events/by-code/%s" 
```
nitpick, but could you please remove this change.

happsie replied:
Sorry, fat fingers. Fixed it, but I will squash the commits as well. So you can wait to trigger a new build 😃

KnutZuidema reviewed (commented):
Thanks for taking this up 👍🏼

happsie commented:
@KnutZuidema I have fixed the formatting issue, also moved to regional routing instead of platform routing for the tournament API. Missed that in the first draft. Feel free to trigger build if you feel like continuing with this change

happsie commented:
@KnutZuidema I can't understand what's wrong with the lint. Do you?

KnutZuidema commented:
@happsie they are broken, sadly. Thanks for this, LGTM 👍🏼
